frontend/website/pages/docs/gui-wallet.mdx

```diff
@@ -11,37 +11,44 @@ We're working hard on the next release of the beta wallet. Stay tuned for update
 
 ![Screenshot of the Coda Wallet GUI.](/static/img/coda-wallet.png)
 
-The GUI Wallet is the easiest way to connect to the Coda network and run a node. You can use the wallet to send transactions, check your funds, stake coda, and create zk-SNARKs to compress transactions.
+The GUI Wallet is the easiest way to connect to the Coda network and run a node. You can use the wallet to send transactions, check your funds, and create zk-SNARKs to compress transactions.
 
 ## Installation
 
-You'll need to make sure TCP ports 8302 and 8303 are open on your router. You can check out our [Getting Started](/docs/getting-started/) guide for more details.
-
-If you have your ports open, the GUI wallet will be able to start Coda for you in the background.
+The Wallet needs to connect to a daemon in order to work, but you will be guided through the proceses of setting up a node when you first launch the wallet. If you already are running a node, you can use that one.
 
 Alternatively, if you already have a node running, the wallet will notice and connect to that running Coda node.
 
 ### MacOS
 
-Stay Tuned!
+[Download the latest build here](https://github.com/CodaProtocol/coda/releases/download/0.1.1-wallet-compat-0.0.11-beta/Coda.Wallet.0.1.1.dmg)
 
-<!-- [Download the latest build here](https://github.com/CodaProtocol/coda/releases/download/0.1.1-wallet-compat-0.0.11-beta/Coda.Wallet.0.1.1.dmg)
+Open the DMG file and drag the wallet into your Applications folder. 
 
-Open the DMG file and drag the wallet into your Applications folder  -->
+The first time you run the wallet you will see this screeen...
 
-### Linux
+Insert here.
 
-Stay Tuned!
+Do not worry!! This is okay, just go to settings and open Security Preferences, then click "Allow" then open Coda and it should start fine.
```

Comment:
haha, we should iron out this part. "You will see this screen. You will need to go into Settings and allow Coda Wallet to be opened."